### PR TITLE
Render descriptions using markdown

### PR DESCRIPTION
**Pull request by ahgittin**
Allows descriptions to use markdown, and it formatted as such, with one-line and multi-line support for cases eg where a type has a very long description.

### CODE REVIEW

**Comment by ahgittin** [comment]
Builds on #195

File: ui-modules/catalog/app/views/catalog/catalog.template.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
         </div>
 
         <div class="col-md-12">
-            <ul class="list-group list-group-bundles" ng-show="state.view === 'bundles'">
+            <ul class="list-group list-group-bundles" ng-if="state.view === 'bundles'">
```
Comment (algairim) [inline ui-modules/catalog/app/views/catalog/catalog.template.html:72]: ng-if removes the whole thing from the DOM tree, I tried it - it is much slower, I would keep ng-show for performance reasons. State can be switched by click, does not make sense to modify DOM tree on every switch.

File: ui-modules/catalog/app/views/catalog/catalog.template.html [hunk 2]
```diff
@@ -165,7 +176,7 @@ <h4>No results matching current filters</h4>
                 </li>
             </ul>
 
-            <ul class="list-group list-group-types" ng-show="state.view === 'types'">
+            <ul class="list-group list-group-types" ng-if="state.view === 'types'">
```
Comment (algairim) [inline ui-modules/catalog/app/views/catalog/catalog.template.html:179]: Same comment applies here, see above.

**Review by algairim — APPROVED** [review]

**Review by algairim — APPROVED** [review]

**Comment by algairim** [comment]
I did not know that "Approve" reply approves the whole pull request. I would consider UI performance impact in catalog view switch.

File: ui-modules/catalog/app/views/catalog/catalog.template.html (hunk 1 above)
Comment (ahgittin) [reply ui-modules/catalog/app/views/catalog/catalog.template.html:72]: If you have a lot of bundles with a lot of types, we _don't_ want them all to render as that is expensive on first load.  I think taking the hit when expanded/collapsed is better.

Comment (algairim) [reply ui-modules/catalog/app/views/catalog/catalog.template.html:72]: I see, makes sense.

**Pull request closed without merging**